src/main/resources/webapp/collection/document/documentation.ts

```diff
@@ -263,7 +263,7 @@ export function respondWithHtml({
 					</tr>
 					<tr>
 						<th>authorization</th>
-						<td>Explorer-Api-Key XXXX</td>
```

Comment:
This is a header value, not a header, so it doesn't need to be lowercased for http/2?